### ISSUE TITLE
Should you update Splat to 2.0.0 ?

### ISSUE THREAD

**Issue by nsulikowski** [reactiveui/ReactiveUI#1344]
Unable to resolve dependencies. 'Splat 2.0.0' is not compatible with 'reactiveui-core 7.3.0 constraint: Splat (>= 1.4.0 && < 2.0.0)

**Comment by jonstodle** [comment]
Since Splat 2.0 is on netstandard (1.1), this could be considered for v8 of RxUI.

**Comment by olevett** [comment]
This is actually already covered under #1332, which is for v8, that  PR is waiting on some more reviewing though.

**Comment by ghuntley** [comment]
2.0 is coming with #1332
